### PR TITLE
feat(footer navigation): Add Footer component

### PR DESCRIPTION
**Pull request by 0xCorgi**
[Current preview](https://corgi-footer.surge.sh/?path=/story/components-menu-footer--default)

image

### CODE REVIEW

File: packages/pancake-uikit/src/components/CakePrice/CakePrice.tsx [hunk 1]
```diff
@@ -21,14 +23,14 @@ const PriceLink = styled.a`
   }
 `;
 
-const CakePrice: React.FC<Props> = ({ cakePriceUsd }) => {
+const CakePrice: React.FC<Props> = ({ cakePriceUsd, color = "textSubtle" }) => {
   return cakePriceUsd ? (
     <PriceLink
-      href="https://pancakeswap.finance/swap?outputCurrency=0x0e09fabb73bd3ade0a17ecc321fd13a19e81ce82"
+      href="https://exchange.pancakeswap.finance/#/swap?outputCurrency=0x0e09fabb73bd3ade0a17ecc321fd13a19e81ce82"
```
Comment (ChefHutch) [inline packages/pancake-uikit/src/components/CakePrice/CakePrice.tsx:29]: This is the old address - the other one is right 👌

File: packages/pancake-uikit/src/components/Svg/Icons/Github.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import Svg from "../Svg";
+import { SvgProps } from "../types";
+
+const Icon: React.FC<SvgProps> = (props) => {
+  return (
+    <Svg xmlns="http://www.w3.org/2000/svg" {...props}>
```
Comment (ChefHutch) [inline packages/pancake-uikit/src/components/Svg/Icons/Github.tsx:7]: I think we ought to declare the viewbox value here. Not sure we need the xmlns attribute? Goes for the other SVGs too

File: packages/pancake-uikit/src/widgets/Menu/icons/Farm.tsx [hunk 3]
```diff
@@ -1,24 +0,0 @@
-import React from "react";
-import Svg from "../../../components/Svg/Svg";
-import { SvgProps } from "../../../components/Svg/types";
-
-const Icon: React.FC<SvgProps> = (props) => {
```
Comment (ChefHutch) [inline packages/pancake-uikit/src/widgets/Menu/icons/Farm.tsx:5]: Awesome to delete all these - can you check they aren't used in the FE somewhere though - some find there way onto one-off cards?

If they are used - move them over to where the regular icons live 👍

**Review by ChefHutch — COMMENTED** [review]
Nice work man, spotted two things on mobile on Brave, this button was really unresponsive:
Screenshot 2021-09-08 at 18 15 51

The menu should presumably be centred, and scale as the device gets smaller? Currently it trails off the screen
Screenshot 2021-09-08 at 18 15 22

File: packages/pancake-uikit/src/widgets/Menu/Menu.tsx [hunk 4]
```diff
@@ -1,15 +1,17 @@
+import { noop } from "lodash";
```
Comment (hachiojidev) [inline packages/pancake-uikit/src/widgets/Menu/Menu.tsx:1]: We can remove this.

**Review by hachiojidev — CHANGES_REQUESTED** [review]
Nice work. Just fixed the small issues Hutch raised and we are good to go.

**Pull request merged**